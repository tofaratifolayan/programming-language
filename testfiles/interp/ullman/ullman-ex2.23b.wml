val pi = 3.1415927 ;
val radius = 4.0 ;
val area = pi *# radius *# radius ;
area ;

