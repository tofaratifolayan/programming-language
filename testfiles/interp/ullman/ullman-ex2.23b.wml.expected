50.2654832
